.github/workflows/continuous-integration.yml

```diff
@@ -26,3 +26,26 @@ jobs:
         run: sudo xargs apt-get install -y < .github/workflows/apt-packages.txt
 
       - run: ci-tests/test-spike
+
+      - name: Calver Release
```

Comment:
Calver is "calendar versioning" but that's not at all obvious to someone reading this

Reply:
fixed